src/GitHub.Exports.Reactive/Services/IGitClient.cs

```diff
@@ -152,7 +152,13 @@ public interface IGitClient
         /// <returns></returns>
         Task UnsetConfig(IRepository repository, string key);
 
-        Task<Remote> GetHttpRemote(IRepository repo, string remote);
+        /// <summary>
+        /// Get a <see cref="Remote"/> that used the http protocol.
+        /// </summary>
+        /// <param name="repository">The repository</param>
+        /// <param name="remote">The name of the remote or null to use original remote.</param>
```

Comment:
It was supposed to find the original remote that was created when the repository was cloned. Alas `libgit2` returns remotes in alphabetical order, so it's impossible to find the first created remote short of manually parsing the `.git\config` file. This would be too much of a hack even for my taste. 😉